src/context/VerifyContext.js

```diff
@@ -154,19 +156,67 @@ export function VerifyProvider(props) {
     setIsLoading(false);
   };
 
+  const getCaptureSelectedArr = () => {
+    return Object.keys(captureImagesSelected)
+      .filter((captureId) => {
+        return captureImagesSelected[captureId] === true;
+      })
+      .map((captureId) => parseInt(captureId));
+  };
+
+  const clickCapture = (payload) => {
+    const { captureId, isShift } = payload;
+    if (isShift) {
+      log.debug('press shift, and there is an anchor:', captureImageAnchor);
+      //if no anchor, then, select from beginning
+      let indexAnchor = 0;
+      if (captureImageAnchor !== undefined) {
+        indexAnchor = captureImages.reduce((a, c, i) => {
+          if (c !== undefined && c.id === captureImageAnchor) {
+            return i;
+          } else {
+            return a;
+          }
+        }, -1);
+      }
+      const indexCurrent = captureImages.reduce((a, c, i) => {
+        if (c !== undefined && c.id === captureId) {
+          return i;
+        } else {
+          return a;
+        }
+      }, -1);
+      let captureShiftSelected = {};
+      captureImages
+        .slice(
+          Math.min(indexAnchor, indexCurrent),
+          Math.max(indexAnchor, indexCurrent) + 1
+        )
+        .forEach((capture) => {
+          captureShiftSelected[capture.id] = true;
+        });
+      setCaptureImagesSelected(captureShiftSelected);
+    } else {
+      setCaptureImagesSelected({
```

Comment:
You need to call `setCaptureImageAnchor(captureId);` here, otherwise shift-click doesn't work properly.